.github/workflows/examples.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
 
     - name: Run example notebooks
       run: |
-        python -m pytest -c pyproject.toml --nbval-lax --dist loadscope -n logical --durations=20 examples/ \
+        python -m pytest -x -c pyproject.toml --nbval-lax --dist loadscope -n logical --durations=20 examples/ \
```

Comment:
This was originally included for debugging but I think it's worth leaving in

Upsides:
* When there is a critical failure, like high-level imports breaking, we get ~100 lines back instead of ~100 bazillion
* Quicker failure reporting when something goes wrong

Downsides:
* Prone to flakiness
* Misses other failures if a change causes multiple